src/Frontend/Core/Engine/TwigTemplate.php

```diff
@@ -39,6 +39,7 @@ public function __construct(
         parent::__construct($environment, $parser, $locator);
 
         $container = Model::getContainer();
+        $this->forkSettings = $container->get('fork.settings');
```

Comment:
Can you then also use $this->forkSettings at the other places where we get it from the container in this class?